test/integration/image-future/default/test/index.test.js

```diff
@@ -992,6 +992,42 @@ function runTests(mode) {
         )
       ).toBe('100%')
     })
+    it('should add position styles to fill images', async () => {
+      expect(
+        await browser.eval(`document.getElementById("fill-image-1").style.left`)
+      ).toBe('0px')
+      expect(
+        await browser.eval(`document.getElementById("fill-image-1").style.top`)
+      ).toBe('0px')
+      expect(
+        await browser.eval(
+          `document.getElementById("fill-image-1").style.right`
+        )
+      ).toBe('0px')
+      expect(
+        await browser.eval(
+          `document.getElementById("fill-image-1").style.bottom`
+        )
+      ).toBe('0px')
+    })
+    it('should not add fill styles to non-fill images', async () => {
+      const fillStyles = [
+        'height',
+        'width',
+        'position',
+        'left',
+        'top',
+        'right',
+        'bottom',
+      ]
+      for (let fillStyle of fillStyles) {
+        expect(
+          await browser.eval(
+            `document.getElementById("non-fill-image").style.${fillStyle}`
+          )
+        ).toBe('')
```

Comment:
I think you can do a single assertion instead of a loop, since non fill should have zero styles.

```js
expect(await browser.elementById('non-fill-image').getAttribute('style')).toMatch('')
```

Reply:
In fact, you could probably add this assertion to the other tests that render the future image without `fill`

Reply:
This test basically covers it

https://github.com/vercel/next.js/blob/59a01ec7090f6c3260d765276db216f3a87af4a2/test/integration/image-future/default/test/index.test.js#L606-L618

Reply:
Sure, makes sense. I removed the test.